src/audio/mixer.cpp

```diff
@@ -438,9 +437,11 @@ void Channel::SetGroup(int group)
 }
 
 Mixer::Mixer()
+	: deviceid(0)
+	, format({ 0 })
+	, effectbuffer(0)
+	, volume(1)
```

Comment:
Fields should be initialised on the line of the definition or within the constructor body. Using this syntax makes it look like those fields are methods when they aren't. Most of the other CPP files is done the way I described.